.github/workflows/ci.yaml

```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on:
+  # Run the CI on any activity on the main branch as well as on any pull
+  # request activity (e.g when it is created and also when it is updated).
+  pull_request:
+  push:
+    branches: [ "main" ]
+
+jobs:
+  build:
+    name: Build and upload SDK (Linux x86-64)
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout seL4 Core Platform repository
+        uses: actions/checkout@v3
+      - name: Checkout seL4 repository
+        uses: actions/checkout@v3
+        with:
+            repository: BreakawayConsulting/seL4
+            ref: sel4cp-core-support
```

Comment:
Wasn't sure whether to have a specific commit hash or to just use the latest commit on `sel4cp-core-support`.